libraries/AP_Baro/AP_Baro_SITL.cpp

```diff
@@ -76,6 +76,15 @@ void AP_Baro_SITL::_timer()
         if (_store_index > _buffer_length - 1) {  // reset buffer index if index greater than size of buffer
             _store_index = 0;
         }
+
+        // if freezed barometer, report altitude to last recorded altitude
+        if (_sitl->baro_freeze[_instance] == 1) {
+            sim_alt = _last_altitude;
+        }
```

Comment:
small formatting issue here, the else should be on the same line as the bracket before it

Reply:
fixed, thanks